.pipelines/ci.yml

```diff
@@ -6,51 +6,71 @@ variables:
 - template: vars.yml
 
 jobs:
-- job: Golang_Code_Lint
-  pool: ARO-CI
+- job: Python_Unit_Tests
+  pool:
+    name: ARO-CI
   steps:
   - template: ./templates/template-checkout.yml
   - script: |
       set -xe
-      make lint-go
-      [[ -z "$(git status -s)" ]]
-    displayName: 🕵️Run Golang Linters
+      make test-python
+    displayName: 🧪Run Python Unit Tests
 
-- job: Python_Unit_Tests
+- job: Check_Code_Generation
   pool:
     name: ARO-CI
   steps:
   - template: ./templates/template-checkout.yml
   - script: |
       set -xe
-      make test-python
+      make generate
       [[ -z "$(git status -s)" ]]
-    displayName: 🧪Run Python Unit Tests
+    displayName: 🕵️Check for missing codegen
+
+- job: Golang_Code_Lint
+  dependsOn: Check_Code_Generation
+  pool: ARO-CI
+  steps:
+  - template: ./templates/template-checkout.yml
+  - script: |
+      set -xe
+      make lint-go
```

Comment:
I think here we should have the [[ -z "$(git status -s)" ]] to check about the effects of linter

Reply:
Ah damn, you are right. I completely forgot that some linters do in-place changes.